### PR TITLE
Add TL files

### PR DESCRIPTION
**Pull request by rlwcomposto**
These files can be used to find lakes in the tangible landscape (TL) using the elevation in the tangible landscape. When the TL is manipulated, these files can be run and the updated lakes will be shown in the TL. The function came from the GRASS GIS wikipage - https://grasswiki.osgeo.org/wiki/Using_GRASS_GIS_through_Python_and_tangible_interfaces_(workshop_at_FOSS4G_NA_2016)#Tangible_Landscape.

### CODE REVIEW

File: activities/RC_TL_v2.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "tasks": [
+    {
+      "layers": [
+        ["d.rast", "map=slope"]
```
[the remark this answers is not in the document]
Comment (rlwcomposto) [reply activities/RC_TL_v2.json]: Thank you for catching that, made the change!

**Comment by petrasovaa** [comment]
Congrats, the checks succeeded! Now try to use your geospatial creativity to customize the copied example. For example:
* display the source point (make it into vector first)
* set the water level dynamically as 50th or 75th percentile of the elevation range
* dynamically find the lowest spot and spread water from there
* something else

**Review by petrasovaa — CHANGES_REQUESTED** [review]
See my last comment.

**Comment by rlwcomposto** [comment]
Hi Anna, sorry for the delay, I didn't realize a change was requested and what that entailed. I just added a little flair to the code  (some colors and rivers). 

I'm running into some consistent errors in the tests that I haven't been able to fix yet.

**Comment by petrasovaa** [comment]
> Hi Anna, sorry for the delay, I didn't realize a change was requested and what that entailed. I just added a little flair to the code (some colors and rivers).
> 
No worries, I think we could do a quick Tangible Landscape session next week.

> I'm running into some consistent errors in the tests that I haven't been able to fix yet.

The relevant error shows incorrect parameter names:

```
ERROR: r.stream.extract: Sorry, <output> is not a valid parameter
ERROR: r.stream.extract: Sorry, <coordinates> is not a valid parameter
ERROR: Required parameter <threshold> not set:
	(Minimum flow accumulation for streams)
```

**Comment by rlwcomposto** [comment]
Changes are complete, the map should display lakes and rivers at the same time in blue.

**Comment by rlwcomposto** [comment]
Mistakenly closed pull request, opening so the code can be reviewed.

**Review by petrasovaa — APPROVED** [review]

**Pull request merged**